ec2net-functions

```diff
@@ -301,6 +291,21 @@ remove_aliases() {
 }
 
 rewrite_aliases() {
+  # Exported by dhclient:
+  local new_ip_address="${new_ip_address}"
+  local reason="${reason}"
+
+  if [[ "$reason" == *6 ]]; then
+      # We're processing a DHCPv6 lease and don't use aliases in the
+      # context of IPv6, so there's nothing to.
```

Comment:
Although the comment is good, logging this would be better to help someone debugging.